tests/test_convert.py

```diff
@@ -129,3 +131,31 @@ class TestConverter:
     )
     def test_from_string(self, name: str, converter: type) -> None:
         assert Converter.from_string(name) == converter
+
+    def test_revealjs_converter(
+        self, tmp_path: Path, presentation_config: PresentationConfig
+    ) -> None:
+        out_file = tmp_path / "slides.html"
+        RevealJS(presentation_configs=[presentation_config]).convert_to(out_file)
+        assert out_file.exists()
+        assert Path(tmp_path / "slides_assets").is_dir()
+        file_contents = Path(out_file).read_text()
+        assert "manim" in file_contents.casefold()
+
+    def test_pdf_converter(
+        self, tmp_path: Path, presentation_config: PresentationConfig
+    ) -> None:
+        out_file = tmp_path / "slides.pdf"
+        PDF(presentation_configs=[presentation_config]).convert_to(out_file)
+        assert out_file.exists()
+
+    def test_pdf_converter_no_presentation_config(self, tmp_path: Path) -> None:
+        with pytest.raises(IndexError):
+            PDF().convert_to(tmp_path / "slides.pdf")
```

Comment:
Actually, nice catch @taiyeoguns! This made me realize that I should disallow having empty lists.

The error is now different:

```suggestion
    def test_converter_no_presentation_config(self) -> None:
        with pytest.raises(ValidationError):
            Converter(presentation_configs=[])
```

And you should add this import line at the top:

```python
from pydantic import ValidationError
```

Reply:
You're welcome @jeertmans, code updated.